tests/Feature/CountryControllerTest.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\Country;
+use Generator;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Inertia\Testing\AssertableInertia as Assert;
+use Tests\TestCase;
+
+class CountryControllerTest extends TestCase
+{
+    use RefreshDatabase;
```

Comment:
Move all `use RefreshDatabase` into `TestCase` and replace with `LazilyRefreshDatabase`.